src/thriveopedia/pages/ThriveopediaHomePage.cs

```diff
@@ -19,4 +33,9 @@ public override void UpdateCurrentWorldDetails()
     public override void OnNavigationPanelSizeChanged(bool collapsed)
     {
     }
+
+    public override void OnTranslationChanged()
+    {
+        content.ExtendedBbcode = TranslationServer.Translate("THRIVEOPEDIA_HOME_INFO");
```

Comment:
We could make it automatically update but it would mean a bigger refactor outside the scope of this PR.

Reply:
I think that would be better to avoid needing to add code in other places just because a custom rich text needs changing. In terms of number of lines of code that needs, I think it'd actually be only about as many as you needed to add right now for this one thing to update...

Reply:
Okay but I won't touch other code that use this similar approach or consider possible performance impact.

Reply:
I was thinking that having a check in the bbcode label to detect if the call to translate did something, if it did then a bool could be set that enables triggering an update based on the translations changed event. That way the performance is not impacted.

Reply:
Here's what I did.
``` csharp
        var old = extendedBbcode;
        var translated = TranslationServer.Translate(extendedBbcode);
        if (translated == old)
            return;
```

Reply:
I guess that is fine enough, I was thinking that even the translate server call could be avoided with my approach as the translations changed callback could just look at the one bool and skip if it isn't set.